src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/model/Address.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.prisonregister.model
+
+import javax.persistence.Entity
+import javax.persistence.FetchType
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+
+@Entity
+data class Address(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long = 0,
```

Comment:
Me and @karenmillermoj had a bit of back and forth on this and reviewed [the Slack thread](https://mojdt.slack.com/archives/CK0QLCQ1G/p1618996314403600)

We went with an integer for this and `PrisonType.id`, rather than `UUID`, because there are no security issues having sequential IDs and clients guessing the next ID because this data is public and not sensitive